task/ps_modules/ssrs.psm1

```diff
@@ -499,7 +499,7 @@ function GetJsonFolderItems($Folder, [Folder]$Parent = $null)
 
     foreach($dataSet in $folder.dataSets)
     {
-        $ds = [DataSet]::new($dataSet.name, $dataSet.fileName, $dataSet.inheritParentSecurity, $dataSet.hidden)
+        $ds = [DataSet]::new($dataSet.name, $dataSet.fileName, $dataSet.inheritParentSecurity, [System.Convert]::ToBoolean($dataSet.hidden))
```

Comment:
In case we make the value of hidden a boolean (check comments above), then there is no need for explicit cast.